src/bundle/Resources/public/js/CKEditor/custom-attributes/ui/custom-attributes-form-view.js

```diff
@@ -209,8 +209,24 @@ class IbexaCustomAttributesFormView extends View {
         addListToDropdown(labeledDropdown.fieldView, itemsList);
 
         this.listenTo(labeledDropdown.fieldView, 'execute', (event) => {
-            const alreadySelectedValues = labeledDropdown.fieldView.element.value ? labeledDropdown.fieldView.element.value.split(' ') : [];
-            const value = config.multiple ? [...new Set([...alreadySelectedValues, event.source.value])].join(' ') : event.source.value;
+            const previousValue = labeledDropdown.fieldView.element.value;
+
+            let value;
+            if (config.multiple) {
+                value = previousValue ? [...new Set([...previousValue.split(','), event.source.value])].join(',') : event.source.value;
+            } else {
+                ( { value } = event.source );
+            }
+
+            if (previousValue === value) {
+                if (config.multiple) {
+                    const set = new Set([...previousValue.split(',')]);
+                    set.delete(event.source.value);
+                    value = [...set].join(',');
+                } else {
+                    value = null;
+                }
+            }
```

Comment:
```suggestion
            const previousValue = labeledDropdown.fieldView.element.value;
            const {  value: eventValue } = event?.source;
            
            let value = null;
            if (config.multiple) {
                value = previousValue ? [...new Set([...previousValue.split(','), eventValue])].join(',') : eventValue;
            } else {
                 value = eventValue;
            }

            if (previousValue === value) {
                if (config.multiple) {
                    const set = new Set([...previousValue.split(',')]);
                    set.delete(eventValue);
                    value = [...set].join(',');
                } else {
                    value = null;
                }
            }
```

Not sure if wee need option chaining for event.source.value

Reply:
@Gengar-i your solution is throwing errors unfortunately in the CI:

> Unsafe usage of optional chaining. If it short-circuits with 'undefined' the evaluation will throw TypeError